citation.cff

```diff
@@ -0,0 +1,48 @@
+cff-version: 1.2.0
+message: All the code in this repository is MIT licensed. When possible, the data used by this project is licensed using the CC-BY-4.0 license. We include attribution and additional license information for third party datasets, and we request that you also maintain that attribution if using this data.
+title: carbonplan / forest-risks
+abstract: This repository includes our libraries and scripts for mapping forest carbon potential and risks.
+authors:
+  - name: "CarbonPlan Team"
+  - name: "Bill Anderegg (@anderegg)"
+  - name: "Grayson Badgley (@badgley)"
+  - name: "Anna Trugman"
+license: MIT - CC-BY-4.0
+
+preferred-citation:
+  title: "Climate risks to carbon sequestration in US forests"
+  abstrct: "Forests are currently a substantial carbon sink globally. Many climate change mitigation strategies rely on forest preservation and expansion, but the effectiveness of these approaches hinges on forests sequestering carbon for centuries despite anthropogenic climate change. Yet climate-driven disturbances pose critical risks to the long-term stability of forest carbon. We quantify the key climate drivers that fuel wildfire, drought, and insects, for the United States over 1984-2018 and project future disturbance risks over the 21st century. We find that current risks are widespread and projected to increase across different emission scenarios by a factor of 4-14 for fire and 1.3-1.8 for drought and insects. Our results provide insights for carbon cycle modeling, conservation, and climate policy, underscoring the escalating climate risks facing forests and the need for emissions reductions to mitigate climate change.Competing Interest StatementThe authors have declared no competing interest."
+  type: preprint
+  year: 2021
+  journal: bioRxiv
+  URL: https://www.biorxiv.org/content/early/2021/05/13/2021.05.11.443688
+	publisher: Cold Spring Harbor Laboratory
+  doi: 10.1101/2021.05.11.443688
+  authors:
+  - family-names: "Anderegg"
+    given-names: "William R.L."
+  - family-names: "Chegwidden"
+    given-names: "Oriana S."
+  - family-names: "Badgley"
+    given-names: "Grayson"
+  - family-names: "Trugman"
+    given-names: "Anna T."
+  - family-names: "Cullenward"
+    given-names: "Danny"
+  - family-names: "Abatzoglou"
+    given-names: "John T."
+  - family-names: "Hicke"
+    given-names: "Jeffrey A."
+  - family-names: "Freeman"
+    given-names: "Jeremy"
+  - family-names: "Hamman"
+    given-names: "Joseph J."
+
+
+identifiers:
+  - description: "forest-risks (code) Zenodo archive"
+    type: doi
+    value: https://doi.org/10.5281/zenodo.4741328
+  - description: "forest-risks (data) Zenodo archive"
+    type: doi
+    value: https://doi.org/10.5281/zenodo.4741332
```

Comment:
not sure what to do here. is this all we can add, or can we also add a description / authors?

Reply:
Added the author list as well as the description from Zenodo. I think we can add a bunch of fields.

https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md

Reply:
This is super helpful. I don't see `repository` as a key that takes a mapping though. So I think we need to rethink this section a bit. I'll write up some broader thoughts on the way this may go above.